gbus/messages.go

```diff
@@ -81,6 +81,11 @@ func (bm *BusMessage) SetPayload(payload Message) {
 	bm.Payload = payload
 }
 
+//TargetSaga allows sending the message to a specific Saga instance
+func (bm *BusMessage) TargetSaga(sagaID string) {
```

Comment:
Do you want to add to the message interface the TargetSaga?

Reply:
there is no interface for the BusMessage